views/about.lavender

```diff
@@ -38,16 +38,16 @@ body
                 a(href='#') Guide Home
               li.dropdown-header Installation
               li
-                a(href="https://github.com/golavender/lavender", title="GitHub") via Composer
+                a(href="/guide.html#composer", title="GitHub") via Composer
```

Comment:
you should either use or not use leading slashes throughout the site consistently. for the record on the internet it would be more correct to have the leading slashes, but with the way you guys are developing that just doesn't work locally